README.md

```diff
@@ -40,9 +40,108 @@ This is a [Devise](https://github.com/plataformatec/devise) extension to add [Tw
 
 ## Pre-requisites
 
-To use the Authy API you will need a Twilio Account, [sign up for a free Twilio account here](https://www.twilio.com/try-twilio).
 
-Create an [Authy Application in the Twilio console](https://www.twilio.com/console/authy/applications) and take note of the API key.
+To use the Twilio Verify API you will need a Twilio Account, [sign up for a free Twilio account here](https://www.twilio.com/try-twilio).
+
+Create an [Twilio Verify Application in the Twilio console](https://www.twilio.com/console/authy/applications) and take note of the API key.
```

Comment:
This isn't the Twilio Verify console URL; this is the one for creating an Authy application which doesn't exist anymore.